README.md

```diff
@@ -1,70 +1,67 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
+[![Netlify Status](https://api.netlify.com/api/v1/badges/14ab8586-1e4c-4dcf-9627-c5d106a9fd32/deploy-status)](https://app.netlify.com/sites/block-cypher/deploys)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Crypto-tracker
 
-## Available Scripts
+ This is a React app consuming the [Coin Gecko](https://www.coingecko.com/en/api/documentation).
+ Displaying the metrics and statistics of top cryptocurrencies.
```

Comment:
- [ ] Kindly explain a little more about the description of the project.
- [ ] Kindly add some project screenshots here.